From fd54a609630a9527655c9d759d2949cd39ac96e8 Mon Sep 17 00:00:00 2001
From: Paul Harrison <pharrison@mythtv.org>
Date: Tue, 11 Mar 2014 09:57:12 +0000
Subject: [PATCH 09/13] AlbumArtImages::scanForImages: fix the image filename

The filename should have been updated to be the myth url of the image.
---
 mythtv/libs/libmythmetadata/musicmetadata.cpp |   15 ++++++++++++++-
 1 file changed, 14 insertions(+), 1 deletion(-)

diff --git a/mythtv/libs/libmythmetadata/musicmetadata.cpp b/mythtv/libs/libmythmetadata/musicmetadata.cpp
index 7614190..c472ebc 100644
--- a/mythtv/libs/libmythmetadata/musicmetadata.cpp
+++ b/mythtv/libs/libmythmetadata/musicmetadata.cpp
@@ -1818,7 +1818,20 @@ void AlbumArtImages::scanForImages()
         image->imageType = (ImageType) strList[x + 1].toInt();
         image->embedded = (strList[x + 2].toInt() == 1);
         image->description = strList[x + 3];
-        image->filename = strList[x + 4];
+
+        if (image->embedded)
+        {
+            image->filename = gCoreContext->GenMythURL(m_parent->Hostname(), 0,
+                                                       QString("AlbumArt/") + strList[x + 4],
+                                                       "MusicArt");
+        }
+        else
+        {
+            image->filename =  gCoreContext->GenMythURL(m_parent->Hostname(), 0,
+                                                        strList[x + 4],
+                                                        "Music");
+        }
+
         image->hostname = strList[x + 5];
         addImage(image);
     }
-- 
1.7.10.2

